config/application.yml.default

```diff
@@ -362,6 +363,7 @@ development:
   attribute_encryption_key_queue: '[{ "key": "11111111111111111111111111111111" }, { "key": "22222222222222222222222222222222" }]'
   aws_logo_bucket: ''
   component_previews_enabled: true
+  component_previews_embed_frame_ancestors: '["http://localhost:4000"]'
```

Comment:
Will the dev docs be set up to load from the local IdP? Or are we just going to load from the dev deployed previews, even in local development of the developer site?

To the point of, maybe we don't set this, and a developer can still override it in their local `config/application.yml` if needed.

```suggestion
```

Reply:
works for me, I will remove it